WooCommerce/Classes/ViewRelated/ReusableViews/SwiftUI Components/FormattableAmountTextField.swift

```diff
@@ -17,7 +17,9 @@ struct FormattableAmountTextField: View {
             // Hidden input text field
             TextField("", text: $viewModel.textFieldAmountText)
                 .onChange(of: viewModel.textFieldAmountText, perform: viewModel.updateAmount)
-                .focused()
```

Comment:
Are you intentionally removing the auto-focus this field gets from the `.focused()` modifier?

If so, we'll need to update the UI tests to tap on this field so it gets focused. (Right now, the tests that include this field are failing because the field doesn't get focus so the keyboard doesn't appear to enter the amount.)

Alternately, we could keep the `.focused()` modifier and just add the `.onChange(of: focusAmountInput)` modifier on top of it. That way the field still gets auto-focus but the placeholder text is also highlighted.